packages/vuetify/src/index.ts

```diff
@@ -9,12 +9,13 @@ import * as directives from './directives'
 
 // Wrapped install function to register all components by default
 // original install for a-la-carte is re-exported in entry-lib.ts
-export default function install (app: App, args: VuetifyUseOptions) {
-  VuetifyInstall(app, {
+export default function install (app: App, preset: GlobalVuetifyPreset = {}) {
+  const args = Object.assign({
```

Comment:
This is the same but worse? Object assign isn't type-safe, and a single use-variable is pointless

Reply:
Need some guidance on how to fix the typings then (if they can be fixed)